.circleci/config.yml

```diff
@@ -90,20 +93,14 @@ orbs:
     jobs:
       checkout:
         <<: *job_defaults
-        parameters:
-          ruby_version:
-            description: Ruby version
-            type: string
-          image:
-            description: Docker image location
-            type: string
         docker:
-          - <<: *container_base
-            image: <<parameters.image>>
+          - image: alpine/git:v2.26.2 # Minimal image with git available
+            user: root # We currently require root to write to `/app`
+        resource_class: small
         steps:
           - checkout
           - save_cache:
-              key: '{{ .Environment.CIRCLE_CACHE_VERSION }}-repo-<<parameters.ruby_version>>-{{ .Environment.CIRCLE_SHA1 }}'
+              key: '{{ .Environment.CIRCLE_CACHE_VERSION }}-repo-{{ .Environment.CIRCLE_SHA1 }}'
```

Comment:
With only 1 checkout step, we don't parametrize our cached key by version anymore.